src/olympia/amo/tests/test_commands.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import mock
+from importlib import import_module
+
+from django.conf import settings
+from django.core.management import call_command
+from django.test.utils import override_settings
+
+
+def sample_cron_job(*args):
+    pass
+
+
+@override_settings(
+    CRON_JOBS={'sample_cron_job': 'olympia.amo.tests.test_commands'})
+@mock.patch('olympia.amo.tests.test_commands.sample_cron_job')
+def test_cron_command(_mock):
+    assert _mock.call_count == 0
+    call_command('cron', 'sample_cron_job', 'arg1', 'arg2')
+    assert _mock.call_count == 1
+    _mock.assert_called_with('arg1', 'arg2')
```

Comment:
Also add tests both the 2 possible error cases to improve coverage (cron command called without args, cron command called with unknown cron)